src/_includes/dartpad-exercise-how-to.md

```diff
@@ -0,0 +1,16 @@
+The embedded editors in this codelab have partially completed code snippets.
```

Comment:
Should this really be a separate file? It's currently only used by the cheatsheet codelab.

Reply:
My thinking is that so long as we are using DartPad, we'll be sharing some instructions / introductory language for it. If I'm mistaken, happy to take this out. I put this here to remind us to consider using common DartPad instructions in the future.

Reply:
Nevertheless I've removed it